seed/java-sdk/http-head/src/main/java/com/seed/httpHead/resources/user/AsyncRawUserClient.java

```diff
@@ -56,10 +56,7 @@ public CompletableFuture<SeedHttpHeadHttpResponse<Void>> head(RequestOptions req
             @Override
             public void onResponse(@NotNull Call call, @NotNull Response response) throws IOException {
                 try (ResponseBody responseBody = response.body()) {
-                    if (response.isSuccessful()) {
-                        future.complete(new SeedHttpHeadHttpResponse<>(null, response));
-                        return;
-                    }
+                    if (response.isSuccessful()) {}
```

Comment:
This isn't right - we'll need to retain the original functionality in the raw client.